chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -2594,9 +2594,12 @@ def transactions_1000_fixture(test_wallet: WalletTool, seeded_random: random.Ran
 # if we try to fill the mempool with more than 550, all spends won't
 # necessarily fit in the block, which the test assumes
 @pytest.mark.anyio
-@pytest.mark.parametrize("mempool_size", [1, 2, 50, 100, 300, 400, 550])
-@pytest.mark.parametrize("seed", [0, 1, 2, 3, 4, 5, 6])
```

Comment:
I removed a few seeds and sizes here to keep the test cases within reason, since they double with the `old` parameter.